catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrCatalogProvider.java

```diff
@@ -129,19 +129,6 @@ public SolrCatalogProvider(
         new ProviderSolrMetacardClient(solrClient, adapter, solrFilterDelegateFactory, resolver);
   }
 
-  /**
-   * Convenience constructor that creates a new ddf.catalog.source.solr.DynamicSchemaResolver
-   *
-   * @param solrClient Solr client
-   * @param adapter injected implementation of FilterAdapter
-   */
-  public SolrCatalogProvider(
```

Comment:
Why isn't this needed anymore?

Reply:
The PR would have required a DynamicSchemaResolver to be added to the constructor, but there is already a constructor with that signature.